src/index.html

```diff
@@ -20,8 +20,312 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul>
+          <li class="nav__item">
+            <a
+              class="is-active"
+              href="#apple"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="card-container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="'iMac with Retina 5K display"
```

Comment:
There is a typo in the alt attribute value. The single quote before 'iMac' should be removed.